gui: Introduces swupd config page

reaganlo opened the pull request:
**Please ensure the above *guidelines for contributing* are met.**

Fixes Issue: #405 #407 #455

Changes proposed in this pull request:
- Introduces swupd config page that can be used to set Mirror URL and Auto Updates
- Moved associated common GUI and TUI strings to swupd.go
- Added corresponding translations
- Made the change to display swupd related pages in sequence in the TUI menu.
- Fixed resize issue due to long strings in **User Add** page #455 as part of fixing the same for this page.
- Refactored code to handle closing pages due to a bug introduced in this PR.

mdhorn commented on swupd/swupd.go:
```diff
@@ -412,7 +459,7 @@ func setMirror(swupdArgs []string, t string) (string, error) {
 
 	if urlErr := network.CheckURL(url); urlErr != nil {
-		return "", fmt.Errorf("Server not responding")
+		return "", fmt.Errorf(utils.Locale.Get("Server not responding"))
```
Initially, I had concerns about having these translated in common code as this means the TUI will get translated strings that might now format right for other languages. But looking through the code base we have similar issues elsewhere, so we will not worry about this for now.

mdhorn commented on gui/pages/swupd_config.go:
```diff
@@ -0,0 +1,284 @@
+	}
+
+	if page.model.SwupdMirror != "" {
+		ret += utils.Locale.Get(". Custom mirror set.")
```
I'm not sure that it makes sense to have this leading person here for the previous statement.

reaganlo replied:
Done to have "." as a separate translated string.

reaganlo commented:
@mdhorn As discussed, I've made the following changes:
- Added corresponding translations
- Made the change to display swupd related pages in sequence in the TUI menu.
- Fixed resize issue due to long strings in **User Add** page #455 as part of fixing the same for this page.
- Refactored code to handle closing pages due to a bug introduced in this PR.
